### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by TheUltimateSike** [qnblackcat/uYouPlus#538]
### Device info
- iOS/iPadOS version: 15.7
- Device model: iPad Mini 5
- Sideload tool (Ex: AltStore, Sideloadly, TrollStore,...) Sideloadly
- The specific version of uYouPlus: _**latest** or **newest** is NOT a version number!!_ v17.39.5-2.1

### Describe the bug
Swiping up on the mini-player (ON IPAD) does not maximise it (so it pops up in the top-left corner), rather the animation as if it were trying to go full screen plays (which would normally appear whenever you swipe up on the slightly bigger player in the top left corner.

### How to reproduce the bug
1. Enable iSponsorBlock (any settings)
2. try to swipe up on the mini-player (on iPad)
3. fail

### Crash log (if possible)
_(The crash log can be found in the Settings app > Privacy > Analytics & Improvements > Analytics Data > Youtube...)_
Not applicable

### Are you using the newest version of uYouPlus? If not, why?
_(Developers spend time and effort to fix bugs & add improvements with every release. Why don't you update to the latest version before reporting about an issue???)_
Not applicable, am on latest

### Does the issue happen with the official YouTube from AppStore?
_(Well, YouTube itself is buggy sometimes...)_
Nope, it's caused by iSponsorBlock

### Additional context: 
Not applicable

**Comment by qnblackcat** [comment]
@PoomSmart could you take a look at it?

**Comment by PoomSmart** [comment]
Is this iSponsorBlock issue though?

**Comment by TheUltimateSike** [comment]
@PoomSmart 95% yes, but it could also be some uyou+ conflict

**Comment by TheUltimateSike** [comment]
@PoomSmart @qnblackcat any update?

**Comment by PoomSmart** [comment]
Yes I confirm that this is indeed iSponsorBlock, and specifically this `YTPlayerView` hook:

https://github.com/Galactic-Dev/iSponsorBlock/blob/master/iSponsorBlock.xm#L471

An issue should be created under iSponsorBlock project.
